integr8ly/integreatly-operator
# [INTLY-5109] Create AWS Resource Alerts

### Description 
Jira: https://issues.redhat.com/browse/INTLY-5109

### Verification
 - Run an install of integreatly from this branch
- Ensure `useClusterStorage` is set to false 
- Wait for all RDS and Elasticache instances to come up 
- Navigate to the prometheus console, ensure all relevant alerts are green
